source/services/ManifestFactory.js

```diff
@@ -539,6 +539,22 @@ class ManifestFactory extends libFableServiceProviderBase
 		{
 			tmpSection.CSSClass = tmpRecord['Section CSS'];
 		}
+		if (tmpRecord['Section Show Title'] && (tmpRecord['Section Show Title'] != ''))
+		{
+			switch(tmpRecord['Section Show Title'].toLowerCase())
```

Comment:
Should this `trim()` as well to avoid losing intent if there is a floating space?